<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHMJoueur</class>
 <widget class="QMainWindow" name="IHMJoueur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Jeu des allumettes</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>match.png</normaloff>match.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.494, radius:0.5, fx:0.5, fy:0.5, stop:0.00980392 rgba(51, 197, 68, 255), stop:0.553922 rgba(40, 153, 53, 255), stop:0.558824 rgba(40, 153, 53, 255), stop:1 rgba(33, 126, 44, 255));
color: rgb(255, 255, 255);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="NbrAllumetteAffich">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>731</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 178, 102, 255), stop:0.00490196 rgba(246, 155, 64, 255), stop:0.553922 rgba(222, 140, 58, 255), stop:1 rgba(174, 110, 45, 255));
color: rgb(0, 0, 0);

border-style: outset;
border-width: 2px;
border-color: rgb(0, 0, 0);
border-top-width: 0px;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="UserAVousDeJouer">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>731</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);

border-style: outset;
border-width: 2px;
border-color: rgb(0, 0, 0);
border-bottom-width: 0px;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu Condensed'; font-size:20pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:11pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nbAllumRM">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);

border-style: outset;
border-width: 2px;
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>1</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="OKBut">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>410</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>OK !</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="NBGame">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>510</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);

border-style: outset;
border-width: 2px;
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:17pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:11pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="NBRound">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);

border-style: outset;
border-width: 2px;
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:17pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:11pt; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="JouerAgainBut">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Jouer encore !</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReinitRGBut">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>508</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Reinitialiser</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>891</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>719</width>
       <height>102</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>88</width>
          <height>100</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="joueur1Label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
        <property name="text">
         <string>Joueur 1 :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Joueur1Line">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);

border-style: outset;
border-width: 2px;
border-color: rgb(255, 255, 255);</string>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="joueur2Label">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
        <property name="text">
         <string>Joueur 2 :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Joueur2Line">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);

border-style: outset;
border-width: 2px;
border-color: rgb(255, 255, 255);</string>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>58</width>
          <height>100</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Jouer_But">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.494, radius:0.5, fx:0.5, fy:0.5, stop:0.00980392 rgba(51, 197, 68, 255), stop:0.553922 rgba(40, 153, 53, 255), stop:0.558824 rgba(40, 153, 53, 255), stop:1 rgba(33, 126, 44, 255));
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Jouer !</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>NbrAllumetteAffich</zorder>
   <zorder>UserAVousDeJouer</zorder>
   <zorder>nbAllumRM</zorder>
   <zorder>OKBut</zorder>
   <zorder>NBGame</zorder>
   <zorder>NBRound</zorder>
   <zorder>ReinitRGBut</zorder>
   <zorder>JouerAgainBut</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
